src/components/App.vue

```diff
@@ -124,6 +109,14 @@ onMounted(async () => {
 
   isEnginesReady.value = true;
 
+  store.dispatch("SET_DIALOG_OPEN", {
```

Comment:
ちょっと忘れそうなのでメモ残しときたいかも
```suggestion
  // ダイアログを表示はエンジン起動後
  store.dispatch("SET_DIALOG_OPEN", {
```